presto-native-execution/presto_cpp/main/types/PrestoToVeloxSplit.cpp

```diff
@@ -114,14 +126,55 @@ velox::exec::Split toVeloxSplit(
               : std::optional<std::string>{*entry.second.value});
     }
 
+    std::unordered_map<std::string, std::string> customSplitInfo;
+    customSplitInfo["table_format"] = "hive-iceberg";
+
+    std::vector<velox::connector::hive::iceberg::IcebergDeleteFile> deletes;
+    deletes.reserve(icebergSplit->deletes.size());
+    for (const auto& deleteFile : icebergSplit->deletes) {
+      std::vector<int32_t> equalityFieldIds;
+      equalityFieldIds.reserve(deleteFile.equalityFieldIds.size());
+      for (auto field : deleteFile.equalityFieldIds) {
+        equalityFieldIds.emplace_back(field);
+      }
+
+      std::unordered_map<int32_t, std::string> lowerBounds;
```

Comment:
Can you try `template< class InputIt > unordered_map( InputIt first, InputIt last)`?

Reply:
Done